earth_enterprise/src/fusion/portableglobe/servers/fileunpacker/shared/file_unpacker.h

```diff
@@ -35,16 +35,6 @@ class GlcReader;
  * Class for unpacking files from a single package file.
  */
 class FileUnpacker {
-  /**
```

Comment:
You should add these back in but as deleted and public.  Also should do the same for move operations

Reply:
Misunderstood the TODO. Thanks.

Reply:
Added back is a deleted and public.